src/publishSubmit.ml

```diff
@@ -318,6 +316,11 @@ let submit
   in
   (* pull-request processing *)
   update_mirror root repo ~user ~token target_branch;
+  mirror_dir, user, token
+
```

Comment:
Splitting the two functions here seems a bit odd to me. Why do you need access to the repository without the new changes?

Reply:
Yes, it does look a little odd; let me know how I can improve it. 

I want the *updated* opam-repository directory locally to run the lint checks on, but want to run the lints before committing the changes to the repository.